### DISCUSSION TITLE
can not make a badges from dblp API json

### DISCUSSION THREAD

**Discussion by youngfish42** [badges/shields#8136] [Q&A] [answered]
Thanks to all developers, I really like the work of shields and I use it to organize the information of various open source programs in my [repo](https://github.com/youngfish42/Awesome-Federated-Learning-on-Graph-and-Tabular-Data). I also introduced the work to my friends in [the Chinese community](https://zhuanlan.zhihu.com/p/533850515) and they all loved shields.

Now I have a problem, I want to use shields to count the number of papers obtained by querying the literature, and I use the [API](https://dblp.org/faq/How+to+use+the+dblp+search+API.html) interface provided by dblp search engine to get the returned json file.
But when I use shields, it always prompts me that the url I fill in is invalid. I don't know what I should do?

Below is the link to the API I constructed, as well as the link to the shields widget, which, as you can see, does not display properly.

dblp API
https://dblp.org/search/publ/api?q=federate%20tree|boost|bagging|gbdt|tabular|forest&format=json&h=1000 

I want to get the number of papers, i.e. the `result.hits.@total` field

shields
https://img.shields.io/badge/dynamic/json?label=dblp&query=result.hits.%40total&url=https%3A%2F%2Fdblp.org%2Fsearch%2Fpubl%2Fapi%3Fq%3Dfederate%2520tree%7Cboost%7Cbagging%7Cgbdt%7Ctabular%7Cforest%26format%3Djson%26h%3D1000 

![](https://img.shields.io/badge/dynamic/json?label=dblp&query=result.hits.%40total&url=https%3A%2F%2Fdblp.org%2Fsearch%2Fpubl%2Fapi%3Fq%3Dfederate%2520tree%7Cboost%7Cbagging%7Cgbdt%7Ctabular%7Cforest%26format%3Djson%26h%3D1000)

I look forward to your responses, thank you!

**Comment by chris48s** [comment, marked as answer]
answered in https://github.com/badges/shields/issues/8137#issuecomment-1169043206
